src/main/resources/scripts/categories/concurrent.groovy

```diff
@@ -0,0 +1,33 @@
+import io.github.arrayv.prompts.SortPrompt
+
+SortPrompt.setSortThreadForCategory('Concurrent Sorts', 22) {
+    // Other
+    run FoldSort with 1024.numbers go()
+    run CreaseSort with 1024.numbers go()
+    run MatrixSort with 256.numbers and 0.667.delay
```

Comment:
personally, I don't like the `and` syntax: commas are simpler and remove the need for an ugly `go()`

Reply:
I like the `and` syntax. However, I think I could rework it to use the Oxford comma. Adding Oxford comma support wouldn't fix `go()`, though.

Reply:
Can you elaborate?

Reply:
people using this scripting API still have to learn how to use it regardless, so keep it simple

Reply:
Is it not simple now?

Reply:
it is not:
- this is coding
- it's still unlike english in the lack of commas and capitalization, and the usage of dots
- also, it's very easy to accidentally add or forget `go()`

Reply:
+ Yes, it's coding
+ The Oxford comma is the command between the second-to-last element and the "and"
+ It pops up a warning if you forget it

Reply:
- coding doesn't mean using natural language, it means following a format that is simple and useful
- what
- how? but a good "compiler" should give good warnings anyway, and it's still annoying

Reply:
+ So you want me to remove the `and`? But how could you do the standard builder pattern of adding options before the final building? The current implementation lets you make a `RunSortBuilder`, and then call `go()` multiple times on it. Should I just add another method that adds options without running it? Also, while coding doesn't mean natural language, me and you are programmers, while not all ArrayV users are.
+  Without Oxford comma: "a, b and c". With Oxford comma: "a, b, and c".
+ Using the "closer" API.

Reply:
- I said before that "people using this scripting API still have to learn how to use it regardless". Anyway, your current implementation does let you run it multiple times, *if you don't use `and` syntax*. The usage of English kinda traps how to write this
- ah right, I forgot; ignore the commas part
- oh I see, nice, though only runs at the end and could be annoying in more complicated applications

personally I think the syntax should be like
```groovy
build SomeSort with [numbers: 256, delay: 0.5] run()
```
but I can get how this can be cumbersome, but to keep the `and` syntax would make code that calls `go()` multiple times really confusing. What about two versions of `with`, one that sets settings, and another that does that and runs the sort?

Reply:
How would you differentiate between the two versions of `with`? Would one use the `256.numbers, 0.5.delay` syntax and the other would use the `numbers: 256, delay: 0.5` syntax? Couldn't that get confusing as to which one is which?

Reply:
call them different names

Reply:
Ok, I added that.

Reply:
That's exactly what `run` does now. I added that, and renamed `go` to `run`. Should I rename it back? I think so tbh.

Reply:
your call, I don't have much preference for either

though imo `and` should be removed (for the reasons above and it's inferior `run`), and `with` inconsistently lets you pass in a map; also it and `put`'s definitions are separated from `and` and `run`

Reply:
`put` is private.

Reply:
I mean that they all utilize `put`

Reply:
Yes, but that's not API.

Reply:
I'm not talking about that, and anyway you don't organize them like that; I mean that they are related and are easier to understand when reading the code when put together

Reply:
Is that grouping ok?

Reply:
Is this better?

Reply:
feels kinda hacky merging the overrides for `with` and `go`, unless that's got some other benefit; otherwise it looks good

Reply:
It allows you to use both features at the same time, if you so desire.

Reply:
true 🤔 though personally I feel it kinda mixes things and only see it being used if you had a default options variable, but you could just use two method calls and spread operator exists too

also maybe have `go` return `this` to allow for running multiple times in a single line? idk though

Reply:
Ok, sure